tasks/components/OrganizationMemberList.tsx

```diff
@@ -159,9 +159,14 @@ export const OrganizationMemberList = ({
         rowMappingToCells={dataObject => [
           <div key="member" className={tw('flex flex-row items-center h-12')}>
             <Avatar avatarUrl={dataObject.avatarURL} alt="" size="small"/>
-            <div className={tw('flex flex-col ml-2')}>
-              <Span className={tw('font-bold h-5')}>{dataObject.name}</Span>
-              <Span type="description" className={tw('text-sm')}>{dataObject.email}</Span>
+            <div className={tw('flex flex-col ml-2 max-w-[250px] overflow-hidden')}>
+              <Span className={tw('font-bold truncate')}>{dataObject.name}</Span>
+              <button
+                // TODO update to a copy to clipboard button
+                onClick={() => navigator.clipboard.writeText(dataObject.email)}
```

Comment:
Why did you add this functionality? Mailto link?

Reply:
mail to link sounds better, but we needed something that allows the user to get the full email, when it is cut off on an overflow

Reply:
[This is what I meant with "Mailto link".](https://en.wikipedia.org/wiki/Mailto) Maybe we can expand the mail or using a popover. The onClick seams a little bit hidden.